readme.md

```diff
@@ -271,6 +271,6 @@ jobs:
     - uses: ./.github/actions/playwright-against-pantheon
       with:
         pantheon_ssh_key: ${{ secrets.PANTHEON_SSH_KEY }}
-        terminus_machine_token: ${{ secrets.TERMINUS_MACHINE_TOKEN }}
+        pantheon_machine_token: ${{ secrets.PANTHEON_MACHINE_TOKEN }}
```

Comment:
Shouldn't this be `machine_token` not `pantheon_machine_token`?

Reply:
eh, this is a job (playwright testing) that exists only in my personal repo. I'm not sure if I want to release that as a reusable action. Should we make these example more rudamentary and just `curl` and check for a 200 or for some text in the response?

Maybe it's simplest to just update these param names to match the param names in this actual action